codespell_lib/data/dictionary.txt

```diff
@@ -22684,6 +22684,8 @@ re-negoziations->re-negotiations
 re-negoziator->re-negotiator
 re-negoziators->re-negotiators
 re-realease->re-release
+re-spawining->respawning
+re-spawning->respawning
```

Comment:
It looks like re-spawning is valid too, so perhaps suggest both.

Reply:
Ah, indeed. Updated in ea13c9205bf06e0682bf0298bff80a9b8c8d6a49, thanks a lot.